setup.cfg

```diff
@@ -77,6 +74,8 @@ envision =
     tornado>=5.1.1
     websocket-client>=1.2.1
     ijson>=3.1.4
+examples = 
+    hydra-core
```

Comment:
Need to update list of examples and instructions at `SMARTS/docs` and `SMARTS/README.md` files.